site/en/blog/cross-origin-isolated-hr-timers/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Aligning timers with cross origin isolation restrictions
+authors:
+  - yoavweiss
+  - agektmr
+description: Starting in Chrome 91, the resolution of explicit timers will be restricted to 100 microseconds across platforms without cross-origin isolation.
+date: 2021-04-15
+hero: 'image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/mnoYgEv2E9L6eSThY9G4.jpg'
+alt: A stopwatch
+---
+
+Before version 91,
+[timer resolutions](https://www.w3.org/TR/hr-time/)
+on Chrome were restricted to 5 microseconds on desktop,
+where [site-isolation](https://www.chromium.org/Home/chromium-security/site-isolation) is enabled,
+and to 100 microseconds on Android, where it's not.
+
+Starting from version 91,
+Chrome will be restricting the resolution of explicit [timers](https://www.w3.org/TR/hr-time/)
+(`performance.now()`, `performance.timeOrigin()`,
+and other performance APIs that expose `DOMHighResTimestamps`) to 100 microseconds across platforms.
+By enabling [cross-origin isolation](https://web.dev/coop-coep/),
+websites can relax the restriction to 5 microseconds regardless of platform.
```

Comment:
Just realized that we're not saying that this is following a related spec change: https://github.com/w3c/hr-time/pull/93

Should we?

Reply:
I add this into the latest commit.